tests/password manager/test_about_login_navigation_from_hamburger_menu.py

```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver import Firefox
+
+from modules.browser_object import PanelUi, TabBar
+
```

Comment:
Please add a fixture called `test_case` which returns the test case number (without the C) as a string.